samples/dashboards/demo/minimal/demo.js

```diff
@@ -89,11 +91,44 @@ Dashboards.board('container', {
             xAxis: {
                 type: 'category'
             },
+            yAxis: {
+                title: {
+                    text: 'mcg'
+                },
+                plotLines: [{
+                    value: 900,
+                    zIndex: 20,
```

Comment:
A z index of 20 makes the tooltip render below the plot line. Which looks buggy.